ci-operator/step-registry/agent/post/agent-post-chain.yaml

```diff
@@ -1,9 +1,7 @@
 chain:
   as: agent-post
   steps:
-  - ref: baremetalds-devscripts-gather
+  - chain: baremetalds-ipi-post
```

Comment:
minor point, but I was used to see the custom steps (agent-gather) before the common ones